<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Nav_Btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>244</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>길 안내(Navigation)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Following_Btn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>243</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>추종(Following)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Voice_Btn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>244</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>음성인식(Voice Recognition)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>150</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
